lib/options.json

```diff
@@ -1,5 +1,54 @@
 {
   "type": "object",
+  "definitions": {
+    "StaticObject": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "directory": {
+          "type": "string",
+          "minLength": 1
+        },
+        "staticOptions": {
+          "type": "object"
+        },
+        "publicPath": {
+          "anyOf": [
+            {
+              "type": "string",
+              "minLength": 1
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "minLength": 1
+              },
+              "minItems": 1
+            }
+          ]
+        },
+        "serveIndex": {
+          "type": "boolean"
+        },
+        "watch": {
+          "anyOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "type": "object"
+            }
+          ]
+        }
+      },
+      "required": ["directory"]
```

Comment:
Do we need this to be required? It seems unnecessary and may be confusing if the user is just specifying watch options or something.

Reply:
I'm going to remove this and let it default to `process.cwd()` as usual if not provided, unless you think it is needed

Reply:
Yep, you are right, we don't need it, because you can set only `watchOptions`